chainer/functions/normalization/l2_normalization.py

```diff
@@ -59,13 +57,14 @@ def normalize(x, eps=1e-5, axis=1):
     This function implements L2 normalization on a vector along the given axis.
     No reduction is done along the normalization axis.
 
-    In the case when :obj:`axis=1` and :math:`x` is a vector of dimension
-    :math:`(N, K)`, where :math:`N` and :math:`K` denote mini-batch size and
-    the dimension of the input variable, this function computes an output
-    vector :math:`y` by the following equation:
+    This function computes an output vector :math:`y` by the following
+    equation:
```

Comment:
The original paragraph showed the most typical example.  I'd like to preserve it somewhere in this docs.

Reply:
I reverted to the original formula, and updated some notation to make it clear `x_i` and `y_j` are vectors.